test/instrumentation/run.sh

```diff
@@ -0,0 +1,11 @@
+# /bin/sh
+version=`node -v`
+versionMinusV=`echo $version | cut -c 2-`
+nodeRoot="node-$versionMinusV"
```

Comment:
No, but this code looks ok to me.

If anything, I'd prefer the last command to be an `exec`, so:

```bash
exec node http.test.js $PWD/$nodeRoot
```